tox.ini

```diff
@@ -22,14 +22,14 @@ deps =
 description = run the unit tests with pytest under {basepython}
 setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
 passenv = DATEUTIL_MAY_CHANGE_TZ TOXENV CI TRAVIS TRAVIS_* APPVEYOR APPVEYOR_* CODECOV_* SYSTEM_* AGENT_* BUILD_* TF_BUILD
-commands = python -m pytest {posargs: "{toxinidir}/dateutil/test" "{toxinidir}/docs" --cov-config="{toxinidir}/tox.ini" --cov=dateutil}
+commands = python -m pytest {posargs: "{toxinidir}/src/dateutil/test" "{toxinidir}/docs" --cov-config="{toxinidir}/tox.ini" --cov=dateutil}
```

Comment:
I think this is another case where `{toxinidir}` will work rather than specifying the actual `tests` directory, once `tests` is moved into the repo root.

Reply:
Running without the tests folder being passed in will run all tests discovered in the current path. I have when pytest/black/etc try to work on the files of things like venv. I rather leave it with `tests/`.